src/Service/CloudFront/tests/Unit/Input/CreateInvalidationRequestTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace AsyncAws\CloudFront\Tests\Unit\Input;
+
+use AsyncAws\CloudFront\Input\CreateInvalidationRequest;
+use AsyncAws\CloudFront\ValueObject\InvalidationBatch;
+use AsyncAws\CloudFront\ValueObject\Paths;
+use AsyncAws\Core\Test\TestCase;
+
+class CreateInvalidationRequestTest extends TestCase
+{
+    public function testRequest(): void
+    {
+
+        $input = new CreateInvalidationRequest([
+            'DistributionId' => 'DistributionId',
+            'InvalidationBatch' => new InvalidationBatch([
+                'Paths' => new Paths([
+                    'Quantity' => 1337,
+                    'Items' => ['all/images'],
+                ]),
+                'CallerReference' => 'abc',
+            ]),
+        ]);
+
+        // see https://docs.aws.amazon.com/cloudfront/latest/APIReference/API_CreateInvalidation2019_03_26.html
```

Comment:
URL does not exists

Reply:
It came from the generated code.
I removed it.